files-community/Files
# Code Quality: Preview popup providers

**Resolved / Related Issues**
   Closes #issue...

**Validation**
How did you test these changes?
   1. Install QuickLook
   2. Select file and press space bar
   3. Use arrow keys to switch the preview
   4. Close QuickLook
   5. Press space bar and verify app doesn't crash

**Screenshots (optional)**
Add screenshots here.
